pynest/nest/tests/test_sp/test_growth_curves.py

```diff
@@ -306,15 +307,21 @@ def simulate(self):
         self.se_python = numpy.zeros(
             (len(self.se_integrator), len(self.sim_steps)))
 
-        start = time.clock()
+        if sys.version_info >= (3, 8):
+            start = time.process_time()
+        else:
+            start = time.clock()
```

Comment:
Three questions here: 
1. I might be blind, but I don't see that the `start` variable is used anywhere in the remainder of the test. Maybe we can just delete the code entirely?
2. If I am wrong on 1., the code occurs twice and may be relevant in other places as well (benchmarks, ...). Should we wrap this in a helper function?
3. `process_time()` has been available since Python 3.3. I would suggest to change the test to `>= (3, 3)` therefore.

`process_time()` has been available since P

Reply:
After I made a few test installations, I am happy to simply delete the code. 
If it's needed elsewhere (benchmarks ..) I can't say. 
@jougs do you know more about that?

Reply:
I don't know more, but am always happy if code gets deleted.